terraform-kots-eks-tf-1.x/variables.tf

```diff
@@ -1,37 +1,38 @@
-# If you prefer to not have prompts when running terraform plan, terraform apply
-# provide defaults to the variables listed below
-
-variable "aws_region" {
-  type        = string
-  description = "AWS region to deploy, i.e. us-west-1"
+locals {
+  cluster_name    = var.cluster_name == "" ? "eks-${var.app_slug}" : var.cluster_name
+  app_and_channel = "${var.app_slug}${var.release_channel != "" ? "/" : ""}${var.release_channel}"
+  k8s_namespace   = var.k8s_namespace == "" ? "${var.app_slug}" : var.k8s_namespace
+  kotsadm_fqdn    = var.kotsadm_fqdn == "" ? "${var.app_slug}-kotsadm.${var.route53_zone_name}" : var.kotsadm_fqdn
+  sentry_fqdn     = var.sentry_fqdn == "" ? "${var.app_slug}-sentry.${var.route53_zone_name}" : var.sentry_fqdn
```

Comment:
Shouldn't have `sentry` hardcoded in the fqdn. And maybe the `sentry_fqdn` should be renamed to `app_fqdn`. But this might be better in a separate PR

Reply:
We have `sentry_fqdn` as variable but it could be changed to application and description updated to explain it. Originally this was intended as we'd be using this but makes sense to generalize it as it's public facing.

Reply:
This is a very good idea!!  Really the only thing that makes this a "sentry" app is the license key that you provide.  We should make this as generic as possible.